manifests/generated/operator-csv.yaml.in

```diff
@@ -758,6 +764,7 @@ spec:
           - virtualmachines/restart
           - virtualmachine/addvolume
           - virtualmachine/removevolume
+          - virtualmachine/memorydump
```

Comment:
Should be "virtualmachines/memorydump"

Reply:
if thats true then the add/remove volume ones are also wrong arent them?

Reply:
@mhenriks fixed in the last commit and also the add/remove volume